nodeup/pkg/model/kube_apiserver.go

```diff
@@ -348,7 +348,10 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 	}
 
 	// build the kube-apiserver flags for the service
-	flags, err := flagbuilder.BuildFlagsList(b.Cluster.Spec.KubeAPIServer)
+	spec := b.Cluster.Spec.KubeAPIServer
+	// clear all appended plugins because flag append-admission-plugins does not exist in kube-apiserver
+	spec.AppendAdmissionPlugins = []string{}
```

Comment:
I wonder if there is a better place to put the appendAdmissionPlugins field, since everything else on KubeAPIServer is a direct mapping of passed-on flags. But that could be handled in a separate PR.

cc @mikesplain @rdrgmnzs to see what their thoughts on this one-off field.

Reply:
this is true, it could be in some another place - but where? If it is not located under kubeapiserver in config, it is maybe misleading? Should we have somekind of kubeAPIServer.Custom. structure which is not added to apiserver?

Reply:
hmm actually if I remove https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/componentconfig.go#L269 flag from this row, it is not added at all anymore to apiserver? I need to test that tomorrow, it is much cleaner change

Reply:
Ah yes, good call 👍